main.py

```diff
@@ -156,11 +155,9 @@ def file_paths_to_dict(resume_file: Path | None, plain_text_resume_file: Path) -
 
         return result
 
-def init_browser() -> webdriver.Chrome:
+def init_browser(browser_type: BrowserType = BrowserType.CHROME) -> webdriver.Chrome | webdriver.Firefox:
```

Comment:
remove this unnecessary

Reply:
Do you mean replacing `browser = init_browser(BrowserFactory.get_browser_type())` with something like `browser = BrowserFactory.get_driver(BrowserFactory.get_browser_type())` ?

Reply:
removed, now the browser get initialized by `BrowserFactory.get_browser()` directly